src/main/java/com/whiletrue/tododemo/dto/Task.java

```diff
@@ -0,0 +1,70 @@
+package com.whiletrue.tododemo.dto;
+
+import java.time.LocalDateTime;
+
+public class Task {
+
+    private String name;
+    private String description;
+    private LocalDateTime dueDateTime;
+    private String createdBy;
+    private String assignedTo;
+    private boolean completed;
+
+    public Task(String name, String description, LocalDateTime dueDateTime, String createdBy, String assignedTo, boolean completed) {
+        this.name = name;
+        this.description = description;
+        this.dueDateTime = dueDateTime;
+        this.createdBy = createdBy;
+        this.assignedTo = assignedTo;
+        this.completed = completed;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
```

Comment:
use Lombok instead of manually writing getters and setters.
https://projectlombok.org/